### PR TITLE
UTY-262: Enabling mac support

### PR DESCRIPTION
**Pull request by jessicafalk**
**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
Checking all shell scripts to allow for development on Mac.
Note: The C# Linter is currently not supported on Mac

#### Tests
manually ran all scripts

#### Documentation
WIP

#### Primary reviewers
@jared-improbable

### CODE REVIEW

File: ci/create-windows-image.ps1 [hunk 1]
```diff
@@ -1,278 +0,0 @@
-### This script is a guide for Improbable's internal build machine images.
```
Comment (jamiebrynes7) [inline ci/create-windows-image.ps1]: Why is this deleted?

Comment (jessicafalk) [reply ci/create-windows-image.ps1]: oops, that happened by accident. Will add it back in

File: tools/schema_compiler/macos/proto/improbable/entity_state.proto [hunk 2]
```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2015 Improbable Worlds Ltd. All Rights Reserved.
```
Comment (jared-improbable) [inline tools/schema_compiler/macos/proto/improbable/entity_state.proto]: Can you modify the core sdk download script to clean the mac version of this and `protoc` up?

Comment (jessicafalk) [reply tools/schema_compiler/macos/proto/improbable/entity_state.proto]: done!
